_posts/2015-03-stitch-fix-heart-unix.md

```diff
@@ -0,0 +1,139 @@
+*Proposed Outline:*
+```
+I. Introduction
+II. Table of contents by problem
+III. Each problem
+  A. Title
+    1. Attribution and environment sub-heading
+  B. Problem
+  C. Solution
+    1. Input
+    2. Command
+    3. Explanation
+IV. ~~Closing~~?
```

Comment:
I think you could have a summary at the end and do something like "What are **your** go-to command-line tricks?